admin_manual/release_notes.rst

```diff
@@ -5,9 +5,22 @@ Release Notes
 Changes in 10.0
 ---------------
 
-* Supported PHP versions are 5.6+ and 7.0+. 
-* The upgrade migration test has been removed; see :ref:`migration_test_label`.
-* Requires to use at least desktop client version 2.0 by default.
+* PHP 7.1 support added (supported PHP versions are 5.6 and 7.0+)
+* The upgrade migration test has been removed; see :ref:`migration_test_label`. (Option "--skip-migration-tests" removed from update command)
+* Requires to use the latest desktop client version 2.3
+* Settings design rework: Admin, personal pages and app management are now merged together into a single "Settings" entry
+* Third party apps are not disabled anymore when upgrading
+* DB Migrations for seamless/improved update process added (enables skipping major versions when upgrading after 10.0)
+* Ability to disable users in the user management panel added (users are prevented from logging in but the account stays as it is and can be enabled again) 
```

Comment:
A long time ago when I was a student I read a book about technical writing and I seem to remember that it's better to just use sentences directly instead of putting too much inside parenthesis. Parenthesis feel like adding an extra layer/stack while putting it as a new sentence frees the cognitive load that was allocate for the first sentence.